e2e/scripts/st_set_page_config.py

```diff
@@ -19,6 +19,7 @@
     page_icon=":shark:",
     layout="wide",
     initial_sidebar_state="collapsed",
+    page_description="Stream Lit description",
```

Comment:
extreme nit: this isn't a big deal at all, but there's no space in Streamlit

Reply:
Just a pun using the word "lit". In the test I even made it "LIT description", but I can change them to whatever feels right to you guys.

Reply:
Oh haha, I caught the pun in the docstring, but it was a bit harder to tell if this one was similarly a pun or an added space. Either way it's minor enough to not be a real issue

Reply:
Funnily, there's an open source project for building web components called [Lit](https://lit.dev/). Probably not a big deal if it's in tests only _but nice to know if someone wants to be absolutely picky_.